libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/offroute/OffRouteDetector.java

```diff
@@ -23,6 +23,7 @@
   private Point lastReroutePoint;
   private OffRouteCallback callback;
   private RingBuffer<Integer> distancesAwayFromManeuver = new RingBuffer<>(3);
+  private static final int TWO_POINTS = 2;
```

Comment:
This is duplicated now in `OffRouteDetector` and `NavigationMapRoute`, but since it's two different packages and the duplication is rather small, I prefered to duplicate it.